runtime/common/src/paras_sudo_wrapper.rs

```diff
@@ -58,5 +65,21 @@ decl_module! {
 			runtime_parachains::schedule_para_cleanup::<T>(id);
 			Ok(())
 		}
+
+		/// Send a downward message to the given para.
+		///
+		/// The given parachain should exist and the payload should not exceed the preconfigured size
+		/// `config.max_downward_message_size`.
+		#[weight = (1_000, DispatchClass::Operational)]
+		pub fn sudo_queue_downward_message(origin, id: ParaId, payload: Vec<u8>) -> DispatchResult {
```

Comment:
probably isnt right to use the term "sudo" here. A root call can also come from democracy or other systems which generate a root origin. `root_queue_downward_message` would make more sense but i think `queue_downward_message` would be enough?